ezgui/src/backend_glow.rs

```diff
@@ -164,16 +165,87 @@ impl<'a> GfxCtxInnards<'a> {
 }
 
 // Something that's been sent to the GPU already.
-// TODO Implement Drop; have to keep a reference to gl.
 pub struct Drawable {
-    _vert_buffer: u32,
-    vert_array: u32,
-    _elem_buffer: u32,
+    vert_array: VertexArray,
+    vert_buffer: Buffer,
+    elem_buffer: Buffer,
     num_indices: i32,
+    gl: Rc<glow::Context>,
+}
+
+impl Drop for Drawable {
+    #[inline]
+    fn drop(&mut self) {
+        self.elem_buffer.destroy(&self.gl);
```

Comment:
Alternatively we could have an Rc<glow::Context> on `Buffer` and `VertexArray`, but this gives us a little Rc'ing.

Reply:
Having it on `Drawable` seems preferable; the 2 buffers and array are also a trio, so treating them as one unit that has to be destroyed and holds onto the context makes sense